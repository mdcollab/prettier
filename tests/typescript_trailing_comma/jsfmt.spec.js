run_spec(__dirname, { parser: "typescript", trailingComma: "all" });
