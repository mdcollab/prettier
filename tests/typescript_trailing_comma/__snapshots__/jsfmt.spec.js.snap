// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trailing.ts 1`] = `
export class BaseSingleLevelProfileTargeting<
	T extends ValidSingleLevelProfileNode,
> {
}

enum Enum {
	x = 1,
	y = 2,
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
export class BaseSingleLevelProfileTargeting<
  T extends ValidSingleLevelProfileNode
> {}

enum Enum {
  x = 1,
  y = 2,
}

`;
